pmaweb/templates/_sponsors.html

```diff
@@ -1,99 +1,113 @@
-
-<div class="sp gold">
-  <a href="https://www.ramotion.com/"><img src="{% static 'images/ramotion.png' %}" alt="Ramotion.com" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://megafamous.com/"><img src="{% static 'images/megafamous.png' %}" alt="MegaFamous" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://socialboss.org/"><img src="{% static 'images/socialboss.png' %}" alt="Social Boss" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://quickbookstoolhub.com/"><img src="{% static 'images/quickbookstoolhub.png' %}" alt="Quickbooks Tool Hub" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://buzzoid.com/"><img src="{% static 'images/buzzoid.png' %}" alt="Buy Instagram Followers & Likes" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://twicsy.com/"><img src="{% static 'images/twicsy.png' %}" alt="Buy Instagram Followers, Likes & Views" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://www.famety.com/"><img src="{% static 'images/famety.png' %}" alt="buy real Instagram followers" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://copycopter.ai/"><img src="{% static 'images/copycopter.png' %}" alt="CopyCopter.ai" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://www.idigic.net/"><img src="{% static 'images/idigic.png' %}" alt="Buy Instagram Followers, Likes & Views" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://buycheapestfollowers.com"><img src="{% static 'images/buycheapestfollowers.png' %}" alt="buycheapestfollowers.com" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://bountii.co"><img src="{% static 'images/bountii.png' %}" alt="bountii.co" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://celebian.com"><img src="{% static 'images/celebian.png' %}" alt="celebian.com" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://www.follower24.de/"><img src="{% static 'images/follower24.png' %}" alt="Buy followers, likes and views for Instagram and TikTok" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://views4you.com/"><img src="{% static 'images/views4you.png' %}" alt="Buy Instagram Followers, Likes & Views" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://viralyft.com/"><img src="{% static 'images/viralyft.png' %}" alt="Viralyft" /></a>
-</div>
-
-<div class="sp gold">
-  <a href="https://www.socialboosting.com"><img src="{% static 'images/socialboosting.png' %}" alt="Social Boosting" /></a>
-</div>
-
-{% include '_sponsorship_note.html' %}
-
+  <h3>Diamond Sponsors</h3>
+  <div class="p-3 border rounded text-center">
+    This space is available &mdash; <a href="{% url 'sponsors' %}#process">contact us</a> to get listed here.
+  </div>
+
+  <h3 class="mt-3">Gold Sponsors</h3>
+  <div class="p-3 border rounded">
+    <div class="row {{ row_cols|default:'row-cols-1 row-cols-md-2' }} g-3">
+      <div class="col text-center">
+        <a href="https://www.vapehuset.se" target="_blank" rel="noopener noreferrer">
```

Comment:
this is one of the most critical parts and lines of the website
please remove noreferrer
see bd92a98889bec2e434c54ca95f8625543b36f47f and the commit message